libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -2789,6 +2789,10 @@ def add_apperiph_defaults(f):
 #define AP_STATS_ENABLED 0
 #endif
 
+#ifndef AP_RCPROTOCOL_ENABLED
+#define AP_RCPROTOCOL_ENABLED 0
```

Comment:
why default of 0? 1 might be better

Reply:
This is in the bit of chibios_hwdef.py which spits out configuration for AP_Periph.  I reckon it's a sensible default there :-)

The actual default for general vehicle builds is in `AP_RCProtocol_config.h` and is 1